Solutions/SlashNext/Playbooks/SlashNextWebAccessLogAssessment/deploy.json

```diff
@@ -0,0 +1,819 @@
+{
```

Comment:
The deployment of this playbook gives the following error:

{
    "status": "Failed",
    "error": {
        "code": "DeploymentFailed",
        "message": "At least one resource deployment operation failed. Please list deployment operations for details. Please see https://aka.ms/DeployOperations for usage details.",
        "details": [
            {
                "code": "Conflict",
                "message": "{\r\n  \"Code\": \"Conflict\",\r\n  \"Message\": \"Website with given name Slashnext-azure-function already exists.\",\r\n  \"Target\": null,\r\n  \"Details\": [\r\n    {\r\n      \"Message\": \"Website with given name Slashnext-azure-function already exists.\"\r\n    },\r\n    {\r\n      \"Code\": \"Conflict\"\r\n    },\r\n    {\r\n      \"ErrorEntity\": {\r\n        \"ExtendedCode\": \"54001\",\r\n        \"MessageTemplate\": \"Website with given name {0} already exists.\",\r\n        \"Parameters\": [\r\n          \"Slashnext-azure-function\"\r\n        ],\r\n        \"Code\": \"Conflict\",\r\n        \"Message\": \"Website with given name Slashnext-azure-function already exists.\"\r\n      }\r\n    }\r\n  ],\r\n  \"Innererror\": null\r\n}"
            }
        ]
    }
}

Reply:
This has been fixed!

Reply:
The issue is resolved for individual playbook deployment but the package deployment failed with following error.
{
    "status": "Failed",
    "error": {
        "code": "DeploymentFailed",
        "message": "At least one resource deployment operation failed. Please list deployment operations for details. Please see https://aka.ms/DeployOperations for usage details.",
        "details": [
            {
                "code": "BadRequest",
                "message": "{\r\n  \"error\": {\r\n    \"code\": \"FunctionNotFound\",\r\n    \"message\": \"The function '/subscriptions/4383ac89-7cd1-48c1-8061-b0b3c5ccfd97/resourceGroups/ankisentinelrg/providers/Microsoft.Web/sites/slashnext-azure-function-ootzm/functions/processlogs' could not be found.\"\r\n  }\r\n}"
            }
        ]
    }
}

Reply:
At the moment we are not able to reproduce mentioned error in our azure environment. We used the same package playbook and it deployed in our environment without any error. Can you please guide us what steps we can follow to reproduce this issue so we can provide the fix. Thanks